Makefile

```diff
@@ -1,229 +1,147 @@
-# $ cd model2owl
-#
-# To see the content of the Makefile, please use the command below
-# $ less Makefile
-#
-########################################################################################################
-#curBranch := $(shell git branch --show-current)
-curBranch := $(shell git rev-parse --abbrev-ref HEAD)
-# Number of file under .git repo (to test if the git repo was already cloned)
-gitRepoFiles :=  $(shell ls -l transform/model2owl/.git >/dev/null 2>&1 | wc -l )
-secrets.GITHUB_TOKEN := *******
-
 # Model2owl directory
-MODEL2OWL_DIR?="."
-# Project directory, ie. directory where we store the input/output from model2owl
-PROJECT_DIR?=".."
-# One of the xmi input files to be merged into a single UML xmi/xml file
-FIRST_INPUT_XMI_FILE?="test/test-multi-xmi/ePO_CM.xml"
-# Input dir containing files to be merged
-INPUT_XMI_DIR?=$(shell dirname ${FIRST_INPUT_XMI_FILE})
+MODEL2OWL_DIR?=.
 # rdflib version
 RDF_LIB_VERSION?=6.2.0
-SAXON?="../saxon-he-10.6.jar"
-# Output directory containing combined file from multiple xmi / xml UML models
-COMBINED_XMI_DIRECTORY?="output/combined-xmi"
-COMBINED_FILE_NAME?=ePO-combined.xmi
-XMI_DIRECTORY?="../output/combined-xmi"
+#Saxon path
+SAXON?=${MODEL2OWL_DIR}/saxon/saxon-he-10.6.jar
 # Glossary output directory
-OUTPUT_GLOSSARY_PATH?="output/glossary"
-
+OUTPUT_GLOSSARY_PATH?=output
+# Convention report output directory
+OUTPUT_CONVENTION_REPORT_PATH?=output
+# Output folder path
+OUTPUT_FOLDER_PATH?=output
 # Input XMI/XML UML file path
-UML_INPUT_FILENAME?="test/test-multi-xmi/ePO_CM.xml"
-filename=$(shell basename -- "${UML_INPUT_FILENAME}") 
-UML_FILENAME_WITHOUT_EXTENSION=$(shell echo ${filename} | cut -f1 -d '.')
-RDF_FILE_PATH?="../file.rdf"
-WIDOCO_OUTPUT_DIRECTORY_NAME?="widoco"
-
-OUTPUT_CORE_FILE_NAME?="ePO_CM-core.rdf"
-OUTPUT_RESTRICTIONS_FILE_NAME?="ePO_CM-restrictions.rdf"
-OUTPUT_SHACL_SHAPES_FILE_NAME?="ePO_CM-shacl.rdf"
-
-# Output ontologies
-OUTPUT_FOLDER_PATH?="../output/epo-ontologies"
-ONTOLOGY_DIR?=${OUTPUT_FOLDER_PATH}
-FILELIST=$(shell ls ${ONTOLOGY_DIR}/*.rdf)
-TURTLE_FILELIST=$(shell ls ${ONTOLOGY_DIR}/*.ttl)
+XMI_INPUT_FILE_PATH?=test/test-multi-xmi/ePO_CM.xml
+#Input filename without extension
+FILENAME=$(shell basename -- "${XMI_INPUT_FILE_PATH}")
+XMI_INPUT_FILENAME_WITHOUT_EXTENSION=$(shell echo ${FILENAME} | cut -f1 -d '.')
+# Output directory containing combined file from multiple xmi / xml UML models
+COMBINED_XMI_DIRECTORY?=output/combined-xmi
+COMBINED_FILE_NAME?=ontologies-combined.xmi
+# Path to one of the xmi input files to be merged into a single UML xmi/xml file
+# All XMIs files to combine need to be in the same directory
+FIRST_INPUT_XMI_FILE_PATH?=test/test-multi-xmi/ePO_CM.xml
+# Input directory containing files to be merged
+INPUT_XMI_DIR?=$(shell dirname ${FIRST_INPUT_XMI_FILE_PATH})
+# Variables for converting in ttl/rdf
+ONTOLOGY_FOLDER_PATH?=${OUTPUT_FOLDER_PATH}
+RDF_FILELIST=$(shell ls ${ONTOLOGY_FOLDER_PATH}/*.rdf)
```

Comment:
I realise now that we have vaiables that belong to different concerns. 

Let's group them nicely what variables are needed for which make targats